backend/pom.xml

```diff
@@ -15,4 +32,13 @@
         <maven.compiler.target>1.7</maven.compiler.target>
```

Comment:
co to jes?
bo jak wersja javy to trochę stara

Reply:
Yeah, could be changed to newer one

Reply:
Actually, it was copy pasted from maven template, i think we can remove that but need to confirm